lib/xeroizer/record/xml_helper.rb

```diff
@@ -107,6 +107,7 @@ def xml_value_from_field(b, field, value)
                 real_value = case value
                   when Date         then value.strftime("%Y-%m-%d")
                   when Time         then value.utc.strftime("%Y-%m-%d")
+                  when String       then Date.parse(value).strftime("%Y-%m-%d")
```

Comment:
This has the potential to silently fail - it would be better to raise an exception and reject the input; so that the user needs to explicitly do this in their code.